client/src/ui/modules/military/battle-view/utils.tsx

```
@@ -40,29 +48,37 @@ function fullStrength(
+  defenderArmy: ArmyBattleInfo | undefined,
   troopConfig: ComponentValue<ClientComponents["TroopConfig"]["schema"]>,
 ) => {
   if (!attackerArmy) return 0;
   const attackingTroops = getTotalTroops(attackerArmy.troops) / configManager.getResourcePrecision();
   return Math.floor(attackingTroops * getChancesOfSuccess(attackerArmy, defenderArmy, troopConfig));
 };
 
-export const getTroopLossOnRaid = (
-  attackerArmy: ArmyInfo | undefined,
-  defenderArmy: ArmyInfo | undefined,
+export const roundDownToPrecision = (value: bigint, precision: Number) => {
```

Biome

[error] Don't use 'Number' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'number' instead


(lint/complexity/noBannedTypes)